handler/pagination.go

```diff
@@ -0,0 +1,30 @@
+package handler
+
+import (
+	"strconv"
+
+	"github.com/ONSdigital/dp-population-types-api/contract"
+)
+
+func manualPagination(limit, offset string, populationTypes []string) (contract.PopulationTypes, int, int) {
+
+	// no need to check for errors as
+	// if non integer, defaults to 0
+	limitInt, _ := strconv.Atoi(limit)
```

Comment:
We already have a queryParams struct in the contract package which is set up to vaidate the values. If you create a GetPopulationTypesRequest that embeds the queryParams as elsewhere you can simply parse the values into the request and they will be validated at parsing time